CircleciLibrary/keywords.py

```diff
@@ -101,7 +104,7 @@ def get_pipeline(self, pipeline_id) -> Pipeline:
         :param pipeline_id: the id of a circleci pipeline
         :return: Pipeline object
         """
-        return Pipeline.from_json(self.api.get_pipeline(pipeline_id))
+        return Pipeline.from_json(trace(self.api.get_pipeline(pipeline_id)))
```

Comment:
@thomasvolk Should the tracing used for debugging this issue be checked into the main repository?

Reply:
Yes, that's the idea. So that we are prepared, if CircleCI will do something funny in the future again ;-)